manager/test/api/src/main/java/io/apiman/manager/test/server/ManagerApiTestServer.java

```diff
@@ -190,18 +198,27 @@ protected void preStart() throws Exception {
             node.start();
             System.out.println("ES node was successfully started.");
 
-            if (!isPersistent) {
-                Node node = NodeBuilder.nodeBuilder().client(true).loadConfigSettings(false)
-                        .clusterName(ES_CLUSTER_NAME).local(true)
-                        .settings(ImmutableSettings.settingsBuilder().build()).node().start();
-                client = node.client();
-            } else {
-                TransportClient tc = new TransportClient(ImmutableSettings.settingsBuilder()
-                        .put("cluster.name", clusterName).build());
-                tc.addTransportAddress(new InetSocketTransportAddress("localhost", 6600));
-                client = tc;
-            }
-
+// Commenting this out for now b/c - the JestClient is of a different Client type 
+//            if (!isPersistent) {
+//                Node node = NodeBuilder.nodeBuilder().client(true).loadConfigSettings(false)
+//                        .clusterName(ES_CLUSTER_NAME).local(true)
+//                        .settings(ImmutableSettings.settingsBuilder().build()).node().start();
+//                client = node.client();
+//            } else {
+//                TransportClient tc = new TransportClient(ImmutableSettings.settingsBuilder()
+//                        .put("cluster.name", clusterName).build());
+//                tc.addTransportAddress(new InetSocketTransportAddress("localhost", 6600));
+//                client = tc;
+//            }
+
+        	String connectionUrl = "http://localhost:6500";
```

Comment:
@EricWittmann @KurtStam For a standalone client like Jest should we have this as a configurable attribute (e.g. apiman.properties) rather than hard-coded?